### PR TITLE
chore: upgrade wp-graphql-cs to `2.0.0-beta`

### PR DESCRIPTION
**Pull request by justlevine**
This PR updates the plugin to use `axepress/wp-graphql-cs:2.0.0-beta`, which is based on WPCS 3.0+.

### CODE REVIEW

**Comment by theodesp** [comment]
Thanks since its a branch of a branch lets review and merge the first branch and then review this branch ;P

**Comment by justlevine** [comment]
> Thanks since its a branch of a branch lets review and merge the first branch and then review this branch ;P

Your call. Did it this way instead of directly as suggestions on #162 because it was faster for me and didn't seem worthy of a separate changeset item, but your call if you prefer to wait until that's merged and then change the base branch here 🤷

**Review by mindctrl — APPROVED** [review]

**Comment by justlevine** [comment]
@theodesp was this closed intentionally, or a side effect of deleting the target branch?

**Comment by theodesp** [comment]
@justlevine probably a side-effect since this action happened automatically. Could you re-submit on a new branch based on the current main please?

**Pull request closed without merging**
